heron/cli/src/python/main.py

```diff
@@ -111,30 +113,21 @@ def run(command, parser, command_args, unknown_args):
   :param unknown_args:
   :return:
   '''
-  status = 1
-  if command == 'activate':
-    status = activate.run(command, parser, command_args, unknown_args)
-
-  elif command == 'deactivate':
-    status = deactivate.run(command, parser, command_args, unknown_args)
-
-  elif command == 'kill':
-    status = kill.run(command, parser, command_args, unknown_args)
-
-  elif command == 'restart':
-    status = restart.run(command, parser, command_args, unknown_args)
-
-  elif command == 'submit':
-    status = submit.run(command, parser, command_args, unknown_args)
-
-  elif command == 'help':
-    status = cli_help.run(command, parser, command_args, unknown_args)
-
-  elif command == 'version':
-    status = version.run(command, parser, command_args, unknown_args)
-
-  return status
-
+  runners = {
+      'activate':activate,
+      'deactivate':deactivate,
+      'kill':kill,
+      'restart':restart,
+      'submit':submit,
+      'update':update,
+      'help':cli_help,
+      'version':version,
+  }
+
+  if command in runners.keys():
+    return runners[command].run(command, parser, command_args, unknown_args)
+  else:
```

Comment:
Yeah, that makes our invocation a bit more fancy, but we still need to maintain both the commands in the parser and the runners dict. I don't think that's a huge deal, but maybe a yak-shave for another day...